src/utils/upload/index.ts

```diff
@@ -120,3 +131,60 @@ export function areFileWithCollectionsSame(
 ): boolean {
     return firstFile.localID === secondFile.localID;
 }
+
+export function analyseUploadFiles(
```

Comment:
What's the purpose of this method? And when/why does is return `NULL_ANALYSIS_RESULT` without doing any anlysis?

Reply:
This function analyses the upload files, suggests the name of the LCA of the files as the collectionName if they are to be uploaded in one collection.

And also returns if the file there are nested folder, in the folder the user is trying to upload.

So for `UPLOAD_TYPE.FILES` it returns NULL_ANALYSIS_RESULT because
1. The browser return path with respect to the LCA ,so for just files,they would have no common ancestor hence we can't suggest a collection name  
2. and there will be no nested folder, as they are just files